docs/getting-started/tutorial-load-dataset.asciidoc

```diff
@@ -151,13 +151,13 @@ The accounts data set doesn't require any mappings, so at this point we're ready
 {ref}/docs-bulk.html[`bulk`] API to load the data sets with the following commands:
 
 [source,shell]
-curl -H 'Content-Type: application/x-ndjson' -XPOST 'localhost:9200/bank/account/_bulk?pretty' --data-binary @accounts.json
-curl -H 'Content-Type: application/x-ndjson' -XPOST 'localhost:9200/shakespeare/doc/_bulk?pretty' --data-binary @shakespeare_6.0.json
-curl -H 'Content-Type: application/x-ndjson' -XPOST 'localhost:9200/_bulk?pretty' --data-binary @logs.jsonl
+curl -u elastic -H 'Content-Type: application/x-ndjson' -XPOST ‘<elasticsearch-URL or localhost>:<port>/<index-name>/<type>_bulk?pretty’ --data-binary @<filename>
```

Comment:
I believe the intention of the format was not to remove specifics of the index/type/filename from these steps, but rather apply the format to the steps. @kellyemurphy 

`curl -u elastic -H 'Content-Type: application/x-ndjson' -XPOST '<elasticsearch-URL or localhost>:9200/bank/account/_bulk?pretty' --data-binary @accounts.json`

Reply:
What was the purpose of changing the single quotes to backticks?

Reply:
Please change it back to single quotes, that was my error.

Reply:
Sure I will change that